src/compiler/prioritization.md

```diff
@@ -1,3 +1,230 @@
+`#t-compiler/wg-prioritization/alerts` Zulip channel][prio_channel].
+
+[prio_channel]: https://rust-lang.zulipchat.com/#narrow/stream/245100-t-compiler.2Fwg-prioritization.2Falerts
+
+To assign a priority, replace the `I-prioritize` label with one of `P-critical`, `P-high`,
+`P-medium` or `P-low` and adding a succinct comment to link the Zulip discussion where the issue
+prioritization occurred, example of a template for the comment:
+
+> WG-prioritization assigning priority ([Zulip discussion](#)).
+>
+> @rustbot label -I-prioritize +P-XXX
+
+Ideally, all [`T-compiler` issues with a `I-prioritize` label][issues_needing_prio] to have a
+priority assigned, or strive to reach this goal: sometimes different factors are blocking issues
+from being assigned a priority label, either because the report or the context is unclear or
+because cannot be reproduced and an MCVE would help. Don't hesitate to ask for clarifications to
+the issue reporter or ping the `ICEbreaker` team when an ICE ("Internal Compiler Errors") needs a
+reduction (add a comment on the issue with `@rustbot ping icebreakers-cleanup-crew`)
+
+Review [stable][stable_regressions], [beta][beta_regressions] and [nightly][nightly_regressions] and
+try to ensure they are assigned when possikle.
+
+[issues_needing_prio]: https://github.com/rust-lang/rust/issues?q=is%3Aopen+is%3Aissue+label%3AT-compiler+-label%3AP-critical+-label%3AP-high+-label%3AP-medium+-label%3AP-low+label%3AI-prioritize
+[stable_regressions]: https://github.com/rust-lang/rust/issues?q=is%3Aopen+label%3Aregression-from-stable-to-stable+-label%3AP-critical+-label%3AP-high+-label%3AP-medium+-label%3AP-low+-label%3AT-infra+-label%3AT-libs+-label%3AT-libs-api+-label%3AT-release+-label%3AT-rustdoc
+[beta_regressions]: https://github.com/rust-lang/rust/issues?q=is%3Aopen+label%3Aregression-from-stable-to-beta+-label%3AP-critical+-label%3AP-high+-label%3AP-medium+-label%3AP-low+-label%3AT-infra+-label%3AT-libs+-label%3AT-libs-api+-label%3AT-release+-label%3AT-rustdoc
+[nightly_regressions]: https://github.com/rust-lang/rust/issues?q=is%3Aopen+label%3Aregression-from-stable-to-nightly+-label%3AP-critical+-label%3AP-high+-label%3AP-medium+-label%3AP-low+-label%3AT-infra+-label%3AT-libs+-label%3AT-libs-api+-label%3AT-release+-label%3AT-rustdoc
+
+The final step prior to generating the agenda is to accept any MCPs. Any MCPs that have had [the `final-comment-period` label][mcp_fcp] 
+for more than ten days can be accepted. Remove the `final-comment-period` label and add the `major-change-accepted` label and then
+close the issue.
+
+[mcp_fcp]: https://github.com/rust-lang/compiler-team/issues?q=is%3Aissue+is%3Aopen+label%3Amajor-change+label%3Afinal-comment-period
+
+Finally, the meeting agenda can be generated. Clone and build [`triagebot`][triagebot] and set two
+environment variables:
+
+- `GITHUB_TOKEN` with a GitHub API token
+- `GOOGLE_API_KEY` with access to the compiler team calendar
```

Comment:
We don't use the Google calendar anymore, I take the chance of this refactor to remove this from the docs :slightly_smiling_face: 

```suggestion
```

Reply:
Fixed